package.json

```diff
@@ -25,11 +25,12 @@
     "@create-figma-plugin/tsconfig": "3.0.2",
     "@create-figma-plugin/ui": "3.0.2",
     "@figma/plugin-typings": "1.79.0",
-    "eslint": "^8.53.0",
+    "eslint": "^8.55.0",
     "preact": "10.18.1",
     "simple-git-hooks": "^2.9.0",
     "typescript": "5.2.2",
-    "vitest": "0.34.6"
+    "vitest": "1.0.4",
+    "xstateV4": "npm:xstate@4"
```

Comment:
Why we have a prefix here?

Reply:
Because the plugin will support also XState V5 in the future 😊

Reply:
I knew there was a file: prefix but I just learnt this, haha, thank you so much.